src/components/Homepage/NewestArticles.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useState } from "react";
+import icon from "../../static/svg/homer.svg";
+import { Thumbnail } from "../Thumbnail/Thumbnail";
+import { Button } from "../Button";
+
+interface IArticle {
+  id: number;
+  heading: string;
+  insight: string;
+  title_photo: string;
+}
+
+export const NewestArticles = () => {
+  const [articles, setArticles] = useState([]);
+  const [loadingError, setLoadingError] = useState(false);
+  useEffect(() => {
+    fetch("http://localhost/api/article/newest")
```

Comment:
don't we have a real endpoint running?

Reply:
also have the given endpoint (`http://localhost/api/`) from a config file imported

Reply:
we need create env file